### PR TITLE
Series perf tests

### PR DESCRIPTION
**Pull request by 1e-to**

### CODE REVIEW

File: sdc/tests/tests_perf/test_perf_series.py [hunk 1]
```diff
@@ -217,11 +217,12 @@ def {func_name}(self):
     ('idxmax', '', [10 ** 9]),
     ('idxmin', '', [10 ** 9]),
     ('isna', '', [2 * 10 ** 7]),
-    ('max', '', [10 ** 9]),
+    ('max', '', [10 ** 8]),
+    ('max', 'skipna=False', [10 ** 8]),
```
Comment (AlexanderKalistratov) [inline sdc/tests/tests_perf/test_perf_series.py]: Shouldn't we also add parameters 'skipna=False' for `mean`, `median`, `idxmax`, `idxmin`, `var`, `cumsum` and so on?
@PokhodenkoSA

File: sdc/tests/tests_perf/test_perf_series.py [hunk 2]
```diff
@@ -234,14 +235,25 @@ def {func_name}(self):
     ('value_counts', '', [3 * 10 ** 5]),
     ('var', '', [5 * 10 ** 8]),
     ('unique', '', [10 ** 5]),
+    ('isnull', '', [10 ** 7]),
+    ('notna', '', [10 ** 7]),
+    ('var', '', [10 ** 8]),
```
Comment (AlexanderKalistratov) [inline sdc/tests/tests_perf/test_perf_series.py]: `var` added twice

File: sdc/tests/tests_perf/test_perf_series.py (hunk 1 above)
Comment (PokhodenkoSA) [reply sdc/tests/tests_perf/test_perf_series.py]: I think we should check that it influence on scaling first. Then we should add this cases to show difference.

File: sdc/tests/tests_perf/test_perf_series.py (hunk 2 above)
Comment (PokhodenkoSA) [reply sdc/tests/tests_perf/test_perf_series.py]: It is better to sort lines alphabetically.

**Pull request merged**